src/components/AnalysisModal.js

```diff
@@ -148,33 +149,34 @@ export const AnalysisModal = withDisplayName('AnalysisModal')(
     const galaxyApp = currentApp(tools.galaxy.label)
     const cromwellApp = currentApp(tools.cromwell.label)
 
+    // TODO: Try to move app/tool-specific info into tools (in notebook-utils.js) so the function below can just iterate over tools instead of duplicating logic
     const renderToolButtons = () => div({
       style: { display: 'flex', alignItems: 'center', flexDirection: 'column', justifyContent: 'space-between' }
     }, [
-      div({
+      h(Clickable, {
         style: styles.toolCard, onClick: () => {
           setCurrentTool(tools.Jupyter.label)
           enterNextViewMode(tools.Jupyter.label)
         }
-      }, [img({ src: jupyterLogoLong, style: _.merge(styles.image, { width: '30%' }) })]),
-      div({
+      }, [img({ src: jupyterLogoLong, alt: 'Create new notebook', style: _.merge(styles.image, { width: '30%' }) })]),
+      h(Clickable, {
         style: styles.toolCard, onClick: () => {
           setCurrentTool(tools.RStudio.label)
           enterNextViewMode(tools.RStudio.label)
         }
-      }, [img({ src: rstudioBioLogo, style: styles.image })]),
-      div({
+      }, [img({ src: rstudioBioLogo, alt: 'Create new R markdown file', style: styles.image })]),
+      h(Clickable, {
         style: { opacity: galaxyApp ? '0.5' : '1', ...styles.toolCard }, onClick: () => {
           setCurrentTool(tools.galaxy.label)
           enterNextViewMode(tools.galaxy.label)
         }, disabled: !galaxyApp, title: galaxyApp ? 'You already have a galaxy environment' : ''
```

Comment:
For some reason the Galaxy and Cromwell buttons are disabled when I have nothing in my Workspace. And there is no indication of **why** they are disabled (the title is not displayed… I'm not sure how it is supposed to be).

If you don't want to fix this bug, please create a ticket for it. I do recall though that these buttons USED to be enabled, so it feels like something odd is going on.

Reply:
This is now working-- thank you!